quil-py/src/program/scheduling.rs

```diff
@@ -0,0 +1,86 @@
+use quil_rs::program::scheduling::{
+    ComputedScheduleItem, FixedSchedule, ScheduledBasicBlock, ScheduledBasicBlockOwned, Seconds,
+    TimeSpan,
+};
+use rigetti_pyo3::{impl_repr, py_wrap_type, pyo3::prelude::*};
+
+use super::PyProgram;
+
+py_wrap_type! {
+    PyScheduledBasicBlock(ScheduledBasicBlockOwned) as "ScheduledBasicBlock"
+}
+
+impl_repr!(PyScheduledBasicBlock);
+
+impl PyScheduledBasicBlock {
+    pub fn as_fixed_schedule(&self, program: &PyProgram) -> PyFixedSchedule {
+        ScheduledBasicBlock::from(&self.0)
```

Comment:
tuple access on the wrapped py types do work, but we typically use the as/into_inner methods. Would like to keep with that pattern for consistency.